.github/workflows/test.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
     - name: Store test results
       if: success() || failure()
-      uses: actions/upload-artifact@v3
+      uses: actions/upload-artifact@v4
```

Comment:
I think we reverted this in 93d00a6b0bbec9850a740ca596d3e9e8d02cb823, did something change in the meantime?

Reply:
The test-reporter workflow is disabled. I'm now working on migrating to v4 for all obs workflows.
The test-reporter fix, will come in a follow-up.